documentation/src/main/asciidoc/userguide/appendices/Configurations.adoc

```diff
@@ -54,6 +54,14 @@ Note that for backwards compatibility, if a https://docs.jboss.org/hibernate/orm
 |`hibernate.connection.isolation` | `REPEATABLE_READ` or
 `Connection.TRANSACTION_REPEATABLE_READ` | Names the JDBC connection transaction isolation level.
 |`hibernate.connection.autocommit` | `true` or `false` (default value) | Names the JDBC connection autocommit mode.
```

Comment:
2 things here...

1. Technically this affects non-DataSource uses as well.  I'd rather call that out here.  As is, this entry is very misleading.
2. There is a synergy between `hibernate.connection.autocommit` and your new setting.  We should discuss that here.